x-pack/test/fleet_api_integration/apis/package_policy/create.ts

```diff
@@ -146,8 +146,8 @@ export default function (providerContext: FtrProviderContext) {
       const { body } = await supertest
         .get(`/internal/saved_objects_tagging/tags/_find?page=1&perPage=10000`)
         .expect(200);
-      expect(body.tags.find((tag: any) => tag.name === 'Managed').relationCount).to.be(6);
-      expect(body.tags.find((tag: any) => tag.name === 'For File Tests').relationCount).to.be(6);
+      expect(body.tags.find((tag: any) => tag.name === 'Managed').relationCount).to.be(9);
```

Comment:
I'm not really sure why this change was necessary. I think it has to do with system created tags, but confirmation on whether or not this change makes sense would be great.

Reply:
We automatically tag saved object installed by Fleet, I guess enabled tags for saved searches mean more object have been tagged